_posts/2020-11-19-Setting-Up-a-Pull-Request.md

```diff
@@ -5,8 +5,9 @@ description:
 summary: 
 tags: 
 ---
-Kajari instructed Celeste and I to start creating a new post using pull request. Pull requests are changes committed in the branch that are prresented to viewers of the users repository on Github. To initiate a new post/pull request, the following commands must be used to complete the task.
-* git checkout -b new post - the purpose of checkout -b is to create a new branch with the name new_post
+Today we started working with Pull Requests. Pull requests are changes committed in the branch that are presented to viewers of the users repository on Github. To initiate a new post/pull request, the following commands must be used to complete the task.
+* git checkout -b new post - the purpose of checkout -b is to create a new branch with the name new_post. The branch can have any name, and is usually a reference to the content of the new post with the information of the user that created the post.
```

Comment:
The branch name `new post` in `git checkout -b new post` cannot be two words. It has to be a single word with either dashes or underscores: `new_post` or `new-post`.